src/app/core/models/music.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { MusicType } from './attribute.types';
+
+export interface Music {
+  id: string;
+  name: string;
+  artist: string;
+  type?: MusicType;
+}
```

Comment:
You manually maintain that and it have to match the firefbase db. Is there a way to generate them with simple tools given by firebase ? Just curious

Reply:
Good point, I'll see if there's an option for that.